content/en/news/releases/1.4.x/announcing-1.4.3/index.md

```diff
@@ -0,0 +1,31 @@
+---
+title: Announcing Istio 1.4.3
+linktitle: 1.4.3
+subtitle: Patch Release
+description: Istio 1.4.3 patch release.
+publishdate: 2020-01-08
+release: 1.4.3
+aliases:
+    - /news/announcing-1.4.3
+---
+
+This release includes bug fixes to improve robustness. This release note describes what’s different between Istio 1.4.2 and Istio 1.4.3.
```

Comment:
Would it make sense to change the first sentence to also mention **Improvments** since we have a separate section below? Maybe `This release includes bug fixes and improvements to improve robustness.` or maybe `This release includes bug fixes to improve robustness as well as some improvements.` I'm not sure I like the double improve* words, and maybe it's _increase_ or _enhance_ robustness.

Reply:
yep, makes sense. I blindly copied this from the last notes